.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea/
-target/
\ No newline at end of file
+target/
+data/demo
```

Comment:
data/demo is the presumed location of the h2 db for you local dev env if you choose to use a file.